wp/providers/githubactions/.ci/.github/workflows/build_deploy_and_test.yml

```diff
@@ -5,22 +5,22 @@ defaults:
     shell: bash
 env:
   TZ: "/usr/share/zoneinfo/America/Los_Angeles"
-  NOTIFY: 'scripts/github/add-commit-comment {project} {sha} "Created multidev environment [{site}#{env}]({dashboard-url})." {site-url}'
```

Comment:
We're not using NOTIFY any more?

Reply:
If I understood correctly, it was passed as the --notify param to build:env:create but that was deprecated a while ago: https://github.com/pantheon-systems/terminus-build-tools-plugin/pull/108/files 

I got asked about this var and started looking and got to the conclusion that it's not used anymore (also, this script doesn't even exist in a d9 site and everything seems to work as expected)

If you confirm that I would be more confident merging this (and with the merge that I just did for d9)